src/services/refactors/refactors.ts

```diff
@@ -4,3 +4,4 @@
 /// <reference path="extractSymbol.ts" />
 /// <reference path="installTypesForPackage.ts" />
 /// <reference path="useDefaultImport.ts" />
+/// <reference path="ConvertToGetterAndSetter.ts" />

```

Comment:
Nit: please keep file casing consistent by renaming the file and this reference to it. (first letter should be lower-case.